modules/dr-restoring-cluster-state.adoc

```diff
@@ -285,6 +285,43 @@ If the status is `Pending`, or the output lists more than one running etcd pod,
 
 .. Repeat this step for each lost control plane host that is not the recovery host.
 
+. Remove the northbound database (nbdb) and southbound database (sbdb). Access the recovery host and the remaining control plane nodes by using Secure Shell (SSH) and run the following command:
++
+[source,terminal]
+----
+$ sudo rm -f /var/lib/ovn/etc/*.db
+----
+
+. Delete all master pods by running the following command:
++
+[source,terminal]
+----
+$ oc delete pods -l app=ovnkube-master -n openshift-ovn-kubernetes
+----
+
+. Ensure that all pods are running by running the following command:
++
+[source,terminal]
+----
+$ oc get  pods -l app=ovnkube-master -n openshift-ovn-kubernetes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                   READY   STATUS    RESTARTS   AGE
+ovnkube-master-nb24h   4/4     Running   0          48s master
+ovnkube-master-rm8kw   4/4     Running   0          47s follower
+ovnkube-master-zbqnh   4/4     Running   0          56s follower
```

Comment:
I don't think we get which `ovnkube-master` pod is the master and which ones are follower from the above `oc get pods` command
```suggestion
ovnkube-master-nb24h   4/4     Running   0          48s
ovnkube-master-rm8kw   4/4     Running   0          47s
ovnkube-master-zbqnh   4/4     Running   0          56s
```

Reply:
Acked